plugin/storage/badger/docs/storage-file-non-root-permission.md

```diff
@@ -0,0 +1,46 @@
+# Badger file permissions as non-root service
```

Comment:
there should be a link to this file from somewhere.

Reply:
Yes, I will open PR at `jaegertracing
/
documentation` after this PR merged.

Like https://github.com/jaegertracing/documentation/pull/685